app/code/community/Bolt/Boltpay/etc/system.xml

```diff
@@ -355,11 +355,20 @@ This function is called when the Bolt checkout modal is closed.
               <show_in_store>0</show_in_store>
               <sort_order>106</sort_order>
             </close>
+            <add_button_everywhere translate="label">
+              <label>Add Bolt Button Everywhere</label>
+              <frontend_type>select</frontend_type>
+              <source_model>adminhtml/system_config_source_yesno</source_model>
```

Comment:
<comment>By default, Bolt checkout button is displayed in cart page. Enable this option if you plan to display Bolt checkout outside of cart page, such as mini cart.</comment>

Reply:
Adde this as a comment to the config